<launch>
	
	<arg name="cam" default="camera2" />
	<arg name="dev_video" default="0" /> 
	<arg name="model" default="logitech" />

	<!-- Start the saver node -->
	<node pkg="record_rgbd_service" type="record_rgb_service" name="$(arg cam)_save_images" output="screen" launch-prefix="">
   	      <param name="camera_name" value="$(arg cam)" />
	      <param name="gscam_config" value="v4l2src device=/dev/video$(arg dev_video) ! video/x-raw-yuv,width=1280,framerate=30/1 ! ffmpegcolorspace  " />
  	</node>
	
	<!-- Save mapping from camera to dev_video -->
	<node name="camera_mapping_$(arg cam)" 
	      pkg="ar_track_service" 
	      type="save_mapping.py"
	      output="screen"
	      args="--dev_video=$(arg dev_video) --cam=$(arg cam)" />

	<!-- Save type -->
	<node name="camera_type_$(arg cam)" 
	      pkg="ar_track_service" 
	      type="save_type.py"
	      output="screen"
	      args="--type=rgb --cam=$(arg cam) --model=$(arg model)" />

	
</launch>
